gc/base/ScanClassesMode.hpp

```diff
@@ -36,23 +36,57 @@ class MM_ScanClassesMode : public MM_Base
 	volatile uintptr_t _scanClassesMode;
 
 public:
+	typedef enum {
```

Comment:
Is scanClassesMode something OMR needs to support? This feels like Java code that should be moved into the language